README.md

```diff
@@ -13,6 +13,8 @@ This repository contains the source code for Zulip's PyPI packages:
 * `zulip_botserver`: [PyPI package](https://pypi.python.org/pypi/zulip-botserver)
   for Zulip's Flask bot server.
 
+The source code is written in *Python 3*.
```

Comment:
While this is a minor update, and is in a separate commit, I would suggest submitting this in a separate PR.

You should be able to cherry-pick the commit into another branch, though it is just one line :)

Reply:
Hmm. I actually *did* mention this in a separate PR: https://github.com/zulip/python-zulip-api/pull/179 but I guess it got carried over to this one too. I will remove it from the PR now.